simple_slurm/scancel.py

```diff
@@ -0,0 +1,56 @@
+import subprocess
+from loguru import logger
+from datetime import datetime, timedelta
+
+class SlurmScancelWrapper:
+
+    sigmtems = {}
+    sigmkills = {}
+    stale_delta = None
+
+    def __init__(self, staledelta = timedelta(minutes=30)):
+        self.stale_delta = staledelta
+
+    def cancel_job(self, job_id):
+        '''Sends a straightforward scancel to a job'''
+        result = subprocess.run(["scancel", job_id],
```

Comment:
I got a TypeError: expected str, bytes or os.PathLike object, not int when using job_id as int.